types/qs/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for qs 6.9
+// Type definitions for qs 7.0
```

Comment:
this doesn’t seem right - qs doesn’t have a v7 yet.

Reply:
I know, but realistically it's a semver break for people using the lib today and I don't want it to magically break builds

Reply:
then let's revert it; i'm going to have a lot more minors of v6 before a v7 comes out, and a version mismatch of a DT package seems like a way worse problem than anything else.

Reply:
(i assumed DT had a different semver policy, since keeping majors and minors in sync is like, the entire purpose of this repo)

Reply:
Works for me 👍